pkg/cli/api/version.go

```diff
@@ -0,0 +1,111 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/Masterminds/semver/v3"
+
+	"github.com/seal-io/walrus/pkg/cli/config"
+	"github.com/seal-io/walrus/utils/log"
+	"github.com/seal-io/walrus/utils/version"
+)
+
+// Info include the client, server and openapi version.
+type Info struct {
+	ClientVersion  Version `json:"clientVersion,omitempty" yaml:"clientVersion,omitempty"`
+	ServerVersion  Version `json:"serverVersion,omitempty" yaml:"serverVersion,omitempty"`
+	OpenAPIVersion Version `json:"openAPIVersion,omitempty" yaml:"-"`
+}
+
+// Version include the version and commit.
+type Version struct {
+	Version      string `json:"version" yaml:"version"`
+	GitCommit    string `json:"gitCommit" yaml:"gitCommit"`
+	IsDevVersion bool   `json:"isDevVersion" yaml:"isDevVersion"`
+}
+
+// GetVersion get client, server and openapi version.
+func GetVersion(sc *config.Config) (*Info, error) {
```

Comment:
The func name is confused, get version but return info

Reply:
rename to VersionInfo